circle.yml

```diff
@@ -0,0 +1,10 @@
+## Customize dependencies
+dependencies:
+  override:
+    - npm prune
```

Comment:
@vdh Sorry completely forgot to answer you
`npm update` is the equivalent of `npm prune` + `npm install`
The only downside is that you lose time by clearing and reinstalling **all** your dependencies

But you are right, greenkeeper is the way to go ;)

Checkout: http://stackoverflow.com/a/19824154/3052444